### PR TITLE
Pep8

### PR DESCRIPTION
**Pull request by RobertoAldea**

### CODE REVIEW

File: code/sys_abs/sys_shd/SYS_SHD.py
Comment (mariuscrsn) [inline code/sys_abs/sys_shd/SYS_SHD.py:1]: Adapt to naming convention

File: code/sys_abs/sys_shd/__init__.py
Comment (mariuscrsn) [inline code/sys_abs/sys_shd/__init__.py:1]: Change naming convention to PEP8

File: code/sys_abs/sys_log/README.md
Comment (mariuscrsn) [inline code/sys_abs/sys_log/README.md:1]: Change to actual template.

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/sys_abs/sys_shd/SYS_SHD.py
Comment (RobertoAldea) [reply code/sys_abs/sys_shd/SYS_SHD.py:1]: Done

File: code/sys_abs/sys_shd/__init__.py
Comment (RobertoAldea) [reply code/sys_abs/sys_shd/__init__.py:1]: Done

File: code/sys_abs/sys_log/README.md
Comment (RobertoAldea) [reply code/sys_abs/sys_log/README.md:1]: Done

File: code/sys_abs/sys_shd/sys_shd.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python3
+"""Copy the attributes of the shared object into the shared object .
+
+Raises:
+    SYS_SHD_Chan_Error_c: [description]
```
Comment (mariuscrsn) [inline code/sys_abs/sys_shd/sys_shd.py:5]: Add description and change exception name

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/sys_abs/sys_shd/sys_shd.py (hunk 1 above)
Comment (RobertoAldea) [reply code/sys_abs/sys_shd/sys_shd.py]: Done

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**